stock_landed_costs_security/readme/USAGE.rst

```diff
@@ -0,0 +1,4 @@
+#. Go to Users.
+#. Select Landing costs -> User.
```

Comment:
```suggestion

```
This is explained in CONFIGURE.rst, isn't it?

Reply:
cc @carolinafernandez-tecnativa